style/extracted_bootstrap_styles.css

```diff
@@ -0,0 +1,2090 @@
+.card {
```

Comment:
This file should be commented where it's from and the license (MIT which matches this repo)